### DISCUSSION TITLE
v14

### DISCUSSION THREAD

**Discussion by MortenDHansen** [owen-it/laravel-auditing#803] [Ideas]
Here is the proposed v14 branch.

https://github.com/owen-it/laravel-auditing/tree/v14-dev

This is the base branch, and apart from a bit of restructuring, it does not (yet) include features and changes. It only cuts support for end-of-life laravel versions, sets a new minimum of PHP 8.0 and removes some deprecated parts.

**Planned features**

- audit table per auditable (see: https://github.com/owen-it/laravel-auditing/discussions/800)
- Change of audit storage paradigm (see: https://github.com/owen-it/laravel-auditing/issues/635, https://github.com/owen-it/laravel-auditing/issues/588 and https://github.com/owen-it/laravel-auditing/discussions/799)
- Update for pivot support
- Let transitionTo() call 'replay' audits so that complete state is restored (see: https://github.com/owen-it/laravel-auditing/issues/754)

**Comment by parallels999** [comment]
Also other deprecations
https://github.com/owen-it/laravel-auditing/search?q=deprecated

**Comment by systemsolutionweb** [comment]
hi, any release date for this? any RC?
